index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <H1>Hello Microverse</H1>    
```

Comment:
- [ ] Your `h` tag is capital. For standard syntax and cleaner code let's please change our

Reply:
Well resolved